product_pricelist_direct_print/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+==============================
+Product Pricelist Direct Print
+==============================
+
+Print price list from menu option, product templates, products variants or
+price lists
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to *Sales > Configuration > Settings*
```

Comment:
Hi @carlosdauden , this is not important but just to say that the first is to install App Sales